lib/portfinder.js

```diff
@@ -77,39 +77,48 @@ exports.getPort = function (options, callback) {
   }
 
   options.host = options.host || '::1';
-  var hosts = [options.host, '0.0.0.0', '127.0.0.1'];
+  var hosts = [options.host, '127.0.0.1', '0.0.0.0'];
```

Comment:
add `::1` to this list if options.host is passed in, as that was a primary goal of #20 - thanks :)